core/src/test/java/org/infinispan/functional/FunctionalInMemoryTest.java

```diff
@@ -55,7 +58,8 @@ public void testWriteLoad(boolean isOwner, WriteMethod method) {
                assertEquals(view.get(), "value");
                return null;
             },
-            (view, nil) -> {}, getClass());
+            (view, nil) -> {
```

Comment:
c'mon, `{}` really does not need to be reformatted.

Reply:
This si the formatter who did that